### DISCUSSION TITLE
Workspace creation is mandatory for oauth implementation???

### DISCUSSION THREAD

**Discussion by vineetkarandikar** [Kong/kong#13804] [Help]
Hii,  
I am facing an issue while getting a service which is deployed publicly, i am not able to access after implementing oauth on service using Kong OSS.
Attached you can find the steps i have taken  i have tried many things but all of them are redirecting me to build a workspace, which demands Enterprise licence.
Please look into the document and guide if i am missing something
[kong_final.docx](https://github.com/user-attachments/files/17554547/kong_final.docx)

**Comment by Oyami-Srk** [comment]
According to [doc](https://docs.konghq.com/hub/kong-inc/oauth2/#endpoints), `oauth2/token` is an endpoint on **proxy port** which by default `8000`. It's not an admin API like `services/`, `routes/`, etc.
